src/wallet/wallet.cpp

```diff
@@ -2163,7 +2163,7 @@ CAmount CWallet::GetBalance(const isminefilter& filter, const int min_depth) con
         for (const auto& entry : mapWallet)
         {
             const CWalletTx* pcoin = &entry.second;
-            if (pcoin->IsTrusted(*locked_chain) && pcoin->GetDepthInMainChain(*locked_chain) >= min_depth) {
+            if (pcoin->IsTrusted(*locked_chain)) {
```

Comment:
This shouldn't count conflicts

Reply:
`IsTrusted` already checks `GetDepthInMainChain > 0`, isn't it enough?